app/main.py

```diff
@@ -0,0 +1,22 @@
+import typing as tp
+import logging
+
+from flask import Flask, jsonify, Response
+
+import settings
+
+logging.basicConfig(format='%(asctime)s - %(levelname)s - %(message)s', level=logging.INFO)
+
+app = Flask(__name__, instance_relative_config=True)
+app.config.from_mapping(
+    SECRET_KEY=settings.SECRET_KEY
```

Comment:
you are not feeding you config from settings.py here

Reply:
can you explain?

Reply:
I see you've removed the file. I mean't you didn't use your settings here, but used some hardcoded values.

Reply:
done